pycodestyle.py

```diff
@@ -119,6 +119,7 @@
 OPERATOR_REGEX = re.compile(r'(?:[^,\s])(\s*)(?:[-+*/|!<=>%&^]+)(\s*)')
 LAMBDA_REGEX = re.compile(r'\blambda\b')
 HUNK_REGEX = re.compile(r'^@@ -\d+(?:,\d+)? \+(\d+)(?:,(\d+))? @@.*$')
+DUNDER_REGEX = re.compile(r'__([^\s]+)__')
```

Comment:
I think it would it be better to make sure this is an assignment as: `r'__([^\s]+)__ = '`